public/javascript/posts/BookAppointment.js

```diff
@@ -19,6 +19,24 @@ var app = angular.module('BookAppointmentApp');
     ];
 
     $scope.viewRenderWeekAgenda = function(view, element){
+      if(view.type == 'month'){
+        $('.fc-day-number').css('cursor', 'pointer');
+        $('.fc-day').css('cursor', 'pointer');
+        $('.fc-week').css('cursor', 'pointer');
+        $('.fc-week').each(function(){
+          $(this).mouseover(function(){

```

Comment:
Once tooltip is attached to elements then you don't need to handle the open event , but yes you will have to handle the close event though